data/case-studies.yaml

```diff
@@ -0,0 +1,25 @@
+categories:
+  - name: External Case Studies
```

Comment:
Just call it 'Case Studies' for now. We will rename it later when we include the internal one!

Reply:
Okay, i just noticed it on the current ticket.